prepare_GBM_data.R

```diff
@@ -0,0 +1,195 @@
+# Script prepares GBM expression data for use in pipelines
+# For GBM array data: convert sample names and remove duplicate individuals
+# For GBM seq data: filter all seq data for GBM samples only, convert genes IDs
+
+# Steven Foltz July 2021
+
+option_list <- list(
+  optparse::make_option("--seq_input",
+                        default = NULL,
+                        help = "TCGA sequencing expression input file path"),
+  optparse::make_option("--array_input",
+                        default = NULL,
+                        help = "refine.bio microarray expression input file path"),
+  optparse::make_option("--metadata_input",
+                        default = NULL,
+                        help = "refine.bio aggregated metadata JSON file path"),
+  optparse::make_option("--array_output",
+                        default = NULL,
+                        help = "Processed microarray data output file path"),
+  optparse::make_option("--seq_output",
+                        default = NULL,
+                        help = "Processed sequencing data output file path"),
+  optparse::make_option("--clinical_input",
+                        default = NULL,
+                        help = "Clinical information input file path (Excel file)"),
+  optparse::make_option("--clinical_output",
+                        default = NULL,
+                        help = "Clinical information output file path (.tsv)"),
+  optparse::make_option("--overwrite",
+                        action = "store_true",
+                        default = FALSE,
+                        help = "Overwrite existing output files [default: %default]")
+)
+
+opt <- optparse::parse_args(optparse::OptionParser(option_list=option_list))
+source("util/option_functions.R")
+check_options(opt, "prepare_GBM_data.R") # how to get this script name easily?
+
+# load libraries
+suppressMessages(library(tidyverse))
+
+# set options
+tcga_seq_expression_input_filepath <- opt$seq_input
+gbm_array_expression_input_filepath <- opt$array_input
+metadata_json_input_filepath <- opt$metadata_input
+gbm_array_output_filepath <- opt$array_output
+gbm_seq_output_filepath <- opt$seq_output
+clinical_xlxs_input_filepath <- opt$clinical_input
+clinical_xlxs_output_filepath <- opt$clinical_output
```

Comment:
This variable name scared me but then I double-checked that you're writing out a TSV

```suggestion
clinical_tsv_output_filepath <- opt$clinical_output
```

Reply:
Yikes!